### DISCUSSION TITLE
Make 'empty' the default column value

### DISCUSSION THREAD

**Discussion by jesusrp98** [supabase/supabase#6695] [Questions] [answered]
If I try to set the default value of a `varchar` column as `''`, it will use those character as literally the default column value.

Is there a way to make the default varchar column value as 'empty', not 'null'? 

![imagen](https://user-images.githubusercontent.com/8961959/166705519-bb0e1261-2b13-49eb-a84a-16748951ff8a.png)

**Comment by GaryAustin1** [comment, marked as answer]
I think you can use format('') as default value to get empty string in the table UI.
You could also alter table in SQL to set the column with default ''

**Comment by jesusrp98** [comment]
Setting the 'Default value' as `format('')` worked!

Thanks!

**Comment by GaryAustin1** [comment]
Just a warning to others coming along, although the format('') option works initially, if you ever edit that column in the table UI or the table structure in the database UI, it will lose the format('') and turning default to a string ormat('' or something like that. 
Open issue:
https://github.com/supabase/supabase/issues/6636

**Comment by stefann01** [comment]
You can set it as an empty string from the Supabase UI:

<img width="1282" height="568" alt="image" src="https://github.com/user-attachments/assets/c179af92-ad52-4e01-9025-9aaccd8808fe" />
